cypress/integration/CreateModels/TagAndFrog.spec.js

```diff
@@ -11,59 +11,87 @@ import * as editDialogModal from '../../support/components/EditDialogModal'
 import * as train from '../../support/Train'
 import * as memoryTableComponent from '../../support/components/MemoryTableComponent'
 import * as common from '../../support/Common'
+import * as helpers from '../../support/Helpers'
 
-describe('CreateModels', () => {
-  it('Tag And Frog', () => {
-    // models.ImportModel('z-tagAndFrog', 'z-tagAndFrog.cl')
-    models.CreateNewModel('z-tagAndFrog')
-    entities.CreateNewEntity({ name: 'multi', multiValued: true })
-    actions.CreateNewActionThenVerifyInGrid({ response: "Hello" })
-    actions.CreateNewActionThenVerifyInGrid({ response: "Hi" })
+describe('Tag And Frog - Create Model', () => {
+  afterEach(helpers.SkipRemainingTestsOfSuiteIfFailed)
+  
+  context('Create', () => {
+    it('Create a model to test against', () => {
+      // models.ImportModel('z-tagAndFrog', 'z-tagAndFrog.cl')
+      models.CreateNewModel('z-tagAndFrog')
+    })
 
-    modelPage.NavigateToTrainDialogs()
-    cy.WaitForTrainingStatusCompleted()
-    train.CreateNewTrainDialog()
+    it('Create a Multivalue Entity', () => {
+      entities.CreateNewEntity({ name: 'multi', multiValued: true })
+    })
+
+    it('Create two Actions', () => {
+      actions.CreateNewActionThenVerifyInGrid({ response: "Hello" })
+      actions.CreateNewActionThenVerifyInGrid({ response: "Hi" })
+    })
+  })
+
+  context('Train', () => {
+    it('Create a new Training Dialog', () => {
+      modelPage.NavigateToTrainDialogs()
+      cy.WaitForTrainingStatusCompleted()
+      train.CreateNewTrainDialog()
+    })
 
     // ------------------------------------------------------------------------
     // This block of code should be removed once we determine and fix the cause
     // of: Bug 1901-Automatic Entity Labeling Is NOT Consistent
     // ------------------------------------------------------------------------
+    it('Create a SPECIAL Training Dialog to deal with bug 1901', () => {
```

Comment:
These titles should explain what is being asserted not what is being done.  It's less important how it's done than if it is done correctly.

E.g. "should save the tags and description on the new dialog"

Then in the test you can clearly see how that is true:
```
                cy.get(testSelectors.trainDialogs.scenarios)
                    .should('contain', testData.description)

                cy.get(testSelectors.trainDialogs.tags)
                    .should('contain', testData.tag01)
```